ee/clickhouse/models/person.py

```diff
@@ -113,6 +113,36 @@ def delete_person(
     sync_execute(DELETE_PERSON_BY_ID, data)
 
 
+# counts duplicate distinct IDs in clickhouse for a team broken down by day
+def count_duplicate_distinct_ids_for_team(team_id: Union[str, int]) -> Dict:
+    query_result = sync_execute(
+        """
+        SELECT startdate, count(*)
+        FROM (
+            SELECT distinct_id, count(*) as count, toDate(min(timestamp)) as startdate
+            FROM (
+                SELECT person_id, distinct_id, max(_timestamp) as timestamp
+                FROM person_distinct_id
+                WHERE team_id = %(team_id)s
+                GROUP BY person_id, distinct_id, team_id
+                HAVING max(is_deleted) = 0
+            )
+            GROUP BY distinct_id
+            HAVING count > 1
+        )
+        GROUP BY startdate
+        """,
+        {"team_id": str(team_id)},
+    )
+    result = {}
```

Comment:
In terms of data shape, I would suggest something like below to make it extra clear when doing analysis
```json
{
    "total": 3,
    "details": [
        { }
   ]
}
```

Reply:
makes sense to me

Reply:
Updated so details are no longer needed.